java_gen/templates/custom/OFFlowAddVer15_getActions.java

```diff
@@ -0,0 +1,11 @@
+
+    @Override
+    public List<OFAction> getActions()throws UnsupportedOperationException {
```

Comment:
These `_getActions` methods look identical across versions and messages... and it seems they are also missing for Version 1.4

I was wondering if I could ask you to this up a little further by consolidating the identical code in a shared partial template and including from all the messages where it is used (like it is done for the `setActions` case).